backend/capellacollab/projects/toolmodels/modelsources/git/gitlab/validation.py

```diff
@@ -27,7 +27,7 @@ async def check_pipeline_health(
 
     try:
         await gitlab_interface.get_last_job_run_id_for_git_model(
```

Comment:
The `gitlab_interface` has no member `get_last_job_run_id_for_git_model` anymore. Can you also change the method to run against all Git integrations?